site/content/en/docs/start/_index.md

```diff
@@ -13,6 +13,7 @@ All you need is Docker (or similarly compatible) container or a Virtual Machine
 
 ## What you’ll need
 
+* 2 CPUs or more
```

Comment:
just a nit: should we say 2 CPU cores ? I wonder if some ppl would think they acutally need two real CPUs.

Reply:
Commented above, think we'll use the same words as the official kubernetes installer ?

Reply:
sounds good